tools/utilities/pythonlibs/audio/training/make_featurizer.py

```diff
@@ -72,15 +72,15 @@ def _create_model(sample_rate, window_size, input_buffer_size, filterbank_type,
         last_node = builder.AddFFTNode(ell_model, ell.nodes.PortElements(last_node.GetOutputPort("output")))
 
     if not filterbank_nfft:
-        # default is to just use the output size of the fft node.
-        filterbank_nfft = last_node.GetOutputPort("output").Size()
+        # default is to just use the size of the fft.
+        filterbank_nfft = nfft
```

Comment:
Yeah, this was a legacy bug issue, perhaps we can fix this now without breaking older models.

Reply:
But we'll need to update the audio unit test model...

Reply:
Ah, and this fix is also not complete, there is a code path before there where "nfft" might not be set.

Reply:
By saying update you mean to say that we need to train the model again? With the changes in the featurizer.

Reply:
No, I believe all the new audio models in the gallery are ok.  So it's a good time to fix this.